src/core/HKXWrangler.cpp

```diff
@@ -3070,9 +3098,20 @@ hkRefPtr<hkpShape> HKXWrapper::build_shape(
 		input.m_vertices = to_bound.m_vertices;
 		hkpCreateShapeUtility::createCapsuleShape(input, output);
 		hkpNamedMeshMaterial* material = new hkpNamedMeshMaterial(materials[0]);
-		hkpCapsuleShape* shape = (hkpCapsuleShape*)output.m_shape;
-		hkInertiaTensorComputer::computeCapsuleVolumeMassProperties(shape->getVertex(0), shape->getVertex(1), shape->getRadius(), mass, properties);
-		return handle_output_transform(output, material, properties, shape_root, body, hk_body);
+		if (output.m_shape->getType() == hkpShapeType::HK_SHAPE_CAPSULE)
```

Comment:
Sometimes computeCapsuleVolumeMassProperties actually generates sphere and it crashes.